packages/graphql/src/entities/Query.ts

```diff
@@ -34,4 +27,8 @@ export class Query {
   navigationMenu (args: unknown, ctx: NexusGenTypes['context']): NxsQueryResult<'navigationMenu'> {
     return ctx.navigationMenu
   }
+
+  cloudViewer (args: unknown, ctx: NxsCtx, info: GraphQLResolveInfo): NxsQueryResult<'cloudViewer'> {
```

Comment:
Shouldn't the remote fields always have a return type of `Promise`?

Reply:
Yep, fixed in the latest commit